src/main/java/com/fundation/logic/view/PlayerFrame.java

```diff
@@ -72,28 +75,23 @@ private void initComponent(String path) {
         volumeSlider = new JSlider(JSlider.VERTICAL, 0, 100, volumeLevel);
         player = new EmbeddedMediaPlayerComponent();
         fileToBePlayed = new File(path);
-        buttonListener = new ButtonListener(volumeLevel);
+        buttonListener = new PlayerButtonListener(volumeLevel);
     }
 
     /**
-     * Set all PlayerJFrame components
+     * Set all PlayerFrame components
      */
     private void initSetting() {
         setBounds(new Rectangle(80, 100, 800, 540));
-        setTitle("LogicalPlayer");
-        try {
-            playButton.setIcon(new ImageIcon("resources/Play.png"));
-            stopButton.setIcon(new ImageIcon("resources/Stop.png"));
-            pauseButton.setIcon(new ImageIcon("resources/Pause.png"));
-        } catch (NullPointerException e) {
-            System.out.println("Player icons cannot be found..");
-        }
-
+        setTitle(fileToBePlayed.getName());
         add(bottomPanel, BorderLayout.SOUTH);
+        playButton.setIcon(new ImageIcon("resources/Images/Play.png"));
         playButton.setPreferredSize(new Dimension(35, 35));
         bottomPanel.add(playButton);
+        pauseButton.setIcon(new ImageIcon("resources/Images/Pause.png"));
         pauseButton.setPreferredSize(new Dimension(35, 35));
         bottomPanel.add(pauseButton);
+        stopButton.setIcon(new ImageIcon("resources/Images/Stop.png"));
         stopButton.setPreferredSize(new Dimension(35, 35));
         bottomPanel.add(stopButton);
         progressBar.setPreferredSize(new Dimension(610, 20));
```

Comment:
In the progress bar change the magical numbers.

Reply:
Magical numbers changed to constants